File: game.py
import random

secret_num = random.randint(1, 100)

game_over = True
attempt = 0
message = ""

while game_over :

    attempt += 1
    number = input("Enter a number: ")

    try:
        number(type) == int

    except:
        print("Please Enter a number between 1 to 100.")
        quit()
    

    if number < secret_num:
        print("Secret number is bigger than yours.")

    elif number > secret_num:
        print("Secret number is smaller than yours.")
    
    else:
        game_over = False

if attempt == 1 :
    message = "attempt"

else:
    message = "attempts"    

print("You guess the secret number in " + str(attempt) + " " + message + ".")


    

File: game.js

var secret_number = Math.floor(Math.random() * 100);

var attempt = 0;
var gameWin = false;
var message = "";


while (gameWin != true){

    attempt += 1 ;
    var userInput = window.prompt("Give your number between 0 to 100.");

    if (userInput < secret_number){
    console.log("please type bigger number !");   
    }

    else if (userInput > secret_number){
    console.log("please type smaller number !");
    }

    else {
    console.log("Congrats!");
    gameWin = true
    }
 
}

if (attempt == 1){
    message = "attempt" ;
}
else{
    message = "attempts";

}

console.log("You succeed in " + attempt.toString() + " " + message + ".");



